sql/chunk_trigger.sql

```diff
@@ -0,0 +1,57 @@
+-- Convert a general trigger definition to a create trigger sql command for a
+-- particular table and trigger name.
```

Comment:
I think we have a naming issue if we have both chunk_trigger.sql and chunk_triggers.sql.